### ISSUE TITLE
Image dimension, ByteBuffer size and format don't match.

### ISSUE THREAD

**Issue by zozeei** [Conezi/face_camera#5]
Hello, thank you v for package.
android not working show error Image dimension, ByteBuffer size and format don't match. Please check if the ByteBuffer is in the decalred format.

**Comment by Taohanxu** [comment]
me too

**Comment by makisekuris** [comment]
I have same problem .
Here is error report
```
I/flutter ( 3992): PlatformException(error, Image dimension, ByteBuffer size and format don't match. Please check if the ByteBuffer is in the decalred format., null, java.lang.IllegalArgumentException: Image dimension, ByteBuffer size and format don't match. Please check if the ByteBuffer is in the decalred format.
I/flutter ( 3992): 	at com.google.android.gms.common.internal.Preconditions.checkArgument(com.google.android.gms:play-services-basement@@18.0.0:2)
I/flutter ( 3992): 	at com.google.mlkit.vision.common.InputImage.<init>(com.google.mlkit:vision-common@@17.1.0:8)
I/flutter ( 3992): 	at com.google.mlkit.vision.common.InputImage.fromByteArray(com.google.mlkit:vision-common@@17.1.0:2)
I/flutter ( 3992): 	at com.google_mlkit_commons.InputImageConverter.getInputImageFromData(InputImageConverter.java:36)
I/flutter ( 3992): 	at com.google_mlkit_face_detection.FaceDetector.handleDetection(FaceDetector.java:55)
I/flutter ( 3992): 	at com.google_mlkit_face_detection.FaceDetector.onMethodCall(FaceDetector.java:41)
I/flutter ( 3992): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:262)
I/flutter ( 3992): 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
I/flutter ( 3992): 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$DartMessenger(DartMessenger.java:319)
I/flutter ( 3992): 	at io.flutter.embedding.engine.dart.-$$Lambda$DartMessenger$TsixYUB5E6FpKhMtCSQVHKE89gQ.run(Unknown Source:12)

```

**Comment by husseinibraahim** [comment]
Me too

**Comment by vecharm** [comment]
```
I/flutter ( 2406): PlatformException(error, Image dimension, ByteBuffer size and format don't match. Please check if the ByteBuffer is in the decalred format., null, java.lang.IllegalArgumentException: Image dimension, ByteBuffer size and format don't match. Please check if the ByteBuffer is in the decalred format.
I/flutter ( 2406): 	at com.google.android.gms.common.internal.Preconditions.checkArgument(com.google.android.gms:play-services-basement@@18.0.0:2)
I/flutter ( 2406): 	at com.google.mlkit.vision.common.InputImage.<init>(com.google.mlkit:vision-common@@17.1.0:8)
I/flutter ( 2406): 	at com.google.mlkit.vision.common.InputImage.fromByteArray(com.google.mlkit:vision-common@@17.1.0:2)
I/flutter ( 2406): 	at com.google_mlkit_commons.InputImageConverter.getInputImageFromData(InputImageConverter.java:36)
I/flutter ( 2406): 	at com.google_mlkit_face_detection.FaceDetector.handleDetection(FaceDetector.java:55)
I/flutter ( 2406): 	at com.google_mlkit_face_detection.FaceDetector.onMethodCall(FaceDetector.java:41)
I/flutter ( 2406): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:262)
I/flutter ( 2406): 	at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
I/flutter ( 2406): 	at io.flutter.embedding.engine.dart.DartMessenger.lambda$dispatchMessageToQueue$0$io-flutter-embedding-engine-dart-DartMessenger(DartMessenger.java:319)
I/flutter ( 2406): 	at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source)
I/flutter ( 2406): 	at android.os.Handler.handleCallback(Handler.java:739)
I/flutter ( 2406): 	at android.os.Handler.dispatchMessage(Handler.java:95)
I/flutter ( 2406): 	at android.os.Looper.loop(Looper.java:148)
I/flutter ( 2406): 	at android.app.ActivityThread.main(ActivityThread.java:5417)
I/flutter ( 2406): 	at java.lang.reflect.Method.invoke(Native Method)
I/flutter ( 2406): 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:746)
I/flutter ( 2406): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:636)
I/flutter ( 2406): )
```

**Comment by LakshayDCoder** [comment]
Same here. Can anyone suggest what can we do to get this working or is there any other alternative for a package like this?

**Comment by Conezi** [comment]
Hi guys, sorry for the delay. Thanks for using our plugin and for reporting the issue. Your feedback is invaluable in improving its quality. I'm glad to let you know that the issue has been resolved in the latest version (0.0.7). Please update your installation and feel free to reach out if you have any more questions. Thank you for your support and contribution to the face_camera plugin community.
